products/views.py

```diff
@@ -27,5 +28,48 @@ def get(self, request, product_id):
         }
 
         return JsonResponse({'results': results}, status=200)
-            
 
+class ListingView(View):
```

Comment:
```suggestion
class ProductsView(View):
```
view 이름은 어떤 자원을 활용하는 지를 나타내는 것이 좋습니다. product 상세와 구분하기 위해 Products로 바꾸는 것이 좋을 것 같습니다.

Reply:
오넵! 변경했습니다!